frontend/src/utils/CampUtils.ts

```diff
@@ -249,3 +249,10 @@ export const isMinCampDetailsFilled = (
   }
   return false;
 };
+
+export const adjustTimeToAmPm = (time: string): string => {
```

Comment:
Is this method still needed/relevant if the camp times are always in 24 hour time? I tried it with a test camp on the following url: 
`http://localhost:3000/register/camp/6397f94e2ff11dcda0ba4960` and got this error:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/57159970/209549268-1f449513-e6ea-4b3b-bf37-5140529efd0f.png">

Reply:
Ah they're empty strings, added handling for that edge case -- should just say "Unknown"